chapter04/README.md

```diff
@@ -247,6 +252,5 @@ but since it is not yet part of the graph up the tree, it has no users yet.
 By adding a bogus user we prevent it being mistaken for dead and being killed.
 
 ## References
-<a id="1">[1]</a>
-Click, C. (1995).
+[^1] Click, C. (1995).
```

Comment:
This is missing a colon. It should be `[^1]: Click, C. (1995).` and the next line needs and indent!